backend/src/utils/route-security.ts

```diff
@@ -161,23 +161,22 @@ const handleSecurityOnRouteData = async (
     await requestSecurityGuard(
       fastify,
       request,
-      request.body.metadata.name,
-      request.body.metadata.namespace,
+      request.body.metadata?.name ?? '',
+      request.body.metadata?.namespace ?? '',
```

Comment:
Undo this change.

Reply:
'request.body.metadata' is possibly 'undefined'

from this type definition in `types.ts`

```
export type K8sResourceCommon = {
  metadata?: {
    name?: string;
    namespace?: string;
    generateName?: string;
    uid?: string;
    labels?: { [key: string]: string };
    annotations?: { [key: string]: string };
    creationTimestamp?: Date;
  };
} & K8sResourceBase;
```